index.js

```diff
@@ -12,6 +12,7 @@ const yaml = require('js-yaml');
 const _ = require('lodash');
 const jwt = require('jsonwebtoken');
 const logger = require('screwdriver-logger');
+const amqp = require('amqp-connection-manager');
```

Comment:
does executor need to be aware of this validation queue ? Can't we handle this at worker ? Once worker hands off the build to executor and get's back the build data, can worker initiate the push to validator queue ?

Reply:
My concern is with adding a new dependency to executor. Now it feels like rabbitmq is a mandatory dependency for executor. Architecture wise, we should limit the awareness of rabbitmq at queue-service & workers

Reply:
So my though process is that executor-* should still have a validate function. 
worker will call the validate function to to verification. This way we can build generic validation solutions for executor-k8s, executor-jenkins etc.

Reply:
ok, let me see if we can move this to worker and handle it there